tools/benchmark/cmd/storage.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2015 Nippon Telegraph and Telephone Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"os"
+	"time"
+
+	"github.com/coreos/etcd/Godeps/_workspace/src/github.com/spf13/cobra"
+	"github.com/coreos/etcd/storage"
+)
+
+var (
+	batchInterval int
+	batchLimit    int
+
+	s storage.KV
+)
+
+func initStorage() {
+	s = storage.NewStore("storage-bench", time.Duration(batchInterval), batchLimit)
+	defer os.Remove("storage-bench")
```

Comment:
This defer is misleading. The file gets removed immediately but everything still works because boltDB still has an open fd; the `defer` can be dropped in favor of just the `os.Remove`